charts/topolvm/values.yaml

```diff
@@ -697,9 +697,15 @@ webhook:
   podMutatingWebhook:
     # webhook.podMutatingWebhook.enabled -- Enable Pod MutatingWebhook.
     enabled: false
+    # webhook.podMutatingWebhook.objectSelector -- Labels required on Pods for webhook action.
```

Comment:
As I previously mentioned in https://github.com/topolvm/topolvm/pull/960#issuecomment-2328087818, we'd like to emphasize that manually editing this field requires close attention. Could you add a warning comment?

Reply:
Yes, I've added warning and put reference into README.md